README.md

```diff
@@ -2,34 +2,62 @@
 
 This repository contains smart contracts and front end interfaces for the Astria Shared Sequencer.
 
-### Rough-draft how to
+## Installation
```

Comment:
I don't think this section header is necessary.

Reply:
removed this